lab1/src/task2/Internship.java

```diff
@@ -0,0 +1,53 @@
+package task2;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Internship {
+    private String name;
+    private String minGrade;
+    private List<Student> students = new ArrayList<>();
+
+    public Internship (String _name, String _mingrade, List<Student> _list) {
```

Comment:
Don't use identifiers starting with _ (it's considered bad coding style). If you want to find out more, see https://www.ibm.com/docs/en/zos/2.3.0?topic=tokens-identifiers